pgp-keys-override.list

```diff
@@ -0,0 +1,230 @@
+# syntax:
+# groupId:artifactId:version =  *,any,badSig,noKey,noSig
```

Comment:
what does 
noSig, noKey, badSig mean?
Maybe a comment could help?
`noSig` we don't need a signature